03-transactions.md

```diff
@@ -62,7 +62,7 @@ Most transaction outputs used here are pay-to-witness-script-hash<sup>[BIP141](h
 
 `2 <pubkey1> <pubkey2> 2 OP_CHECKMULTISIG`
 
-* Where `pubkey1` is the numerically lesser of the two DER-encoded `funding_pubkey` and where `pubkey2` is the numerically greater of the two.
+* Where `pubkey1` is the numerically lesser of the two `funding_pubkey` in compressed format, and where `pubkey2` is the numerically greater of the two.
```

Comment:
Not related to this PR but how does one compare compare two byte arrays numerically?

Reply:
What we're doing is byte-per-byte (taking the pubkey as a big-endian number):

```scala
  @tailrec
  def isLessThan(a: ByteVector, b: ByteVector): Boolean = {
    if (a.isEmpty && b.isEmpty) false
    else if (a.isEmpty) true
    else if (b.isEmpty) false
    else if (a.head == b.head) isLessThan(a.tail, b.tail)
    else ((a.head & 0xff) < (b.head & 0xff))
  }
```

That follows https://github.com/bitcoin/bips/blob/master/bip-0069.mediawiki

Is that an issue?

Reply:
The way you're doing this is not an issue. I just wanted to point out that it's not clear from the text how to convert byte arrays to numbers for comparing, i.e., interpreting the entire 33 bytes  as big-endian. Maybe this should be added.

Reply:
True, and probably worth a separate PR. I'll open one.